### PR TITLE
Fix connection issue on China servers

### PR DESCRIPTION
**Pull request by mark-juwai**
## Summary of changes

- Add variable for download url, so that we can override it.

## How to Test

- It's very straightforward.
- Test with this [PR](https://github.com/juwai/mobile-api-v2-ansible/pull/3) if you really want to.

@tjoelsson @agirivera Please review.

### CODE REVIEW

**Comment by stuartwan** [comment]
how does that solve the china connection issue? :)

File: defaults/main.yml [hunk 1]
```diff
@@ -1,2 +1,3 @@
 ---
 zeromq_version: 4.0.5
+zeromq_download_url: https://archive.org/download/zeromq_{{ zeromq_version }}/zeromq-{{ zeromq_version }}.tar.gz
```
Comment (tjoelsson) [inline defaults/main.yml]: Will this trigger a deprecation warning for "bare variables"? Better add quotes around it.

**Comment by mark-juwai** [comment]
>how does that solve the china connection issue? :)

We uploaded the file to China S3: https://s3.cn-north-1.amazonaws.com.cn/juwai-devops/zeromq/zeromq_4.0.5/zeromq-4.0.5.tar.gz

By adding variable for the download url, we can override it in the config file if it's China server.

File: defaults/main.yml (hunk 1 above)
Comment (mark-juwai) [reply defaults/main.yml]: Added.

**Comment by stuartwan** [comment]
👍 but better remove the ticket id from title since it's public repo.

Also, since 4.0.8 zeromq has moved to Github for release download, instead of archive.org. Can consider upgrading to that.

**Review by tjoelsson — APPROVED** [review]

**Review by agirivera — APPROVED** [review]
Looks correct :+1:

**Comment by mark-juwai** [comment]
@imlazyone Thank you for the advises, I've amended the commit message. But concerning the upgrade, I would prefer to fix the issue at hand for now.

Thank you. @tjoelsson @agirivera

**Comment by mark-juwai** [comment]
whoops, I don't have permission to merge it. @tjoelsson Please help to do that. Thanks.

**Pull request merged**